### DISCUSSION TITLE
request: Passwordless Email Auth Domain Whitelabeling

### DISCUSSION THREAD

**Discussion by Animalmix55** [Web3Auth/Web3Auth#1332] [Feature Requests]
Hello!

We are looking to see if Web3Auth has investigated adding the ability to "whitelabel" the passwordless email domain from no-reply@tor.us to a custom email domain. Other products that we use, like ActiveCampaign allow us to leverage a DKIM signature to sign emails that they send on our behalf; this causes emails to show as being sent by our domain rather than theirs.

I ask as, being that we are in Web3, scams are rampant and we feel that having users click links in emails from non-product domains is suspicious and may condition our users into being vulnerable to future email exploits.

I imagine that this is already possible in a self-hosted environment, but I hope there's a solution to be made outside of self-hosting.

- Cory

**Comment by Wgil** [comment]
+1
